guardian/src/main/java/com/auth0/android/guardian/sdk/GuardianAPIClient.java

```diff
@@ -337,9 +337,8 @@ public Response intercept(Chain chain) throws IOException {
                             .header("Accept-Language",
                                     Locale.getDefault().toString())
                             .header("User-Agent",
-                                    String.format("GuardianSDK/%s(%s) Android %s",
```

Comment:
The code was **always** "1". So I removed this line instead of generating another BuildConfig constant